com.valvesoftware.Steam.CompatibilityTool.Proton.yml

```diff
@@ -628,16 +637,19 @@ modules:
       - -I../wine/include
       - -I../openvr/headers
       - -L${FLATPAK_DEST}/lib32
-      - -L${FLATPAK_DEST}/lib32/wine
+      - -L${FLATPAK_DEST}/lib32/wine/i386-unix
       - -lsteam_api
       - -lole32
       - -ldl
       - --guiexe
       - .
     make-args: *winemaker_m32_make_args
+    build-commands:
+      - winebuild --exe --fake-module -F steam -o steam.exe.fake
     no-make-install: true
     post-install:
-      - install -Dm755 steam_helper.exe.so ${FLATPAK_DEST}/lib32/wine/steam.exe.so
+      - install -Dm755 steam_helper.exe.so ${FLATPAK_DEST}/lib32/wine/i386-unix/steam.exe.so
+      - install -Dm755 steam.exe.fake ${FLATPAK_DEST}/lib32/wine/i386-windows/steam.exe
```

Comment:
I'm not sure if this is actually needed. Does the original build ship `steam.exe`?

Reply:
The build from GloriousEggroll ships it, yes. Not sure about the proton builds from Valve.